GenderPayGap.WebUI/Models/Organisation/ManageOrganisationViewModel.cs

```diff
@@ -1,21 +1,169 @@
+        private readonly List<int> yearsWithDraftReturns;
+
+        public ManageOrganisationViewModel(Database.Organisation organisation, User user, List<int> yearsWithDraftReturns)
+        {
+            Organisation = organisation;
+            User = user;
+            this.yearsWithDraftReturns = yearsWithDraftReturns;
+        }
+
+        public List<User> GetFullyRegisteredUsersForOrganisationWithCurrentUserFirst()
+        {
+            List<User> users = Organisation.UserOrganisations
+                .Where(uo => uo.PINConfirmedDate.HasValue)
+                .Select(uo => uo.User)
+                .ToList();
+            
+            // The current user must be in this list (otherwise we wouldn't be able to visit this page)
+            // So, remove the user from wherever they are n the list
+            // And insert them at the start of the list
+            users.Remove(User);
+            users.Insert(0, User);
+            
+            return users;
+        }
+
+        public List<ManageOrganisationDetailsForYearViewModel> GetOrganisationDetailsForYears()
+        {
+            List<int> reportingYears = ReportingYearsHelper.GetReportingYears();
+            var detailsForYears = new List<ManageOrganisationDetailsForYearViewModel>();
+
+            foreach (int reportingYear in reportingYears)
+            {
+                // Get organisation's scope for given reporting year
+                var scopeForYear = Organisation.GetScopeForYear(reportingYear);
+
+                if (scopeForYear != null)
+                {
+                    detailsForYears.Add(new ManageOrganisationDetailsForYearViewModel(
+                        Organisation,
+                        reportingYear,
+                        yearsWithDraftReturns.Contains(reportingYear)
+                    ));
+                }
+            }
+
+            return detailsForYears;
+        }
+
+    }
+
+    public class ManageOrganisationDetailsForYearViewModel
+    {
+        public int ReportingYear { get; }
+
+        private readonly Database.Organisation organisation;
+        private readonly bool hasDraftReturnForYear;
+
+        public ManageOrganisationDetailsForYearViewModel(Database.Organisation organisation, int reportingYear, bool hasDraftReturnForYear)
+        {
+            this.organisation = organisation;
+            ReportingYear = reportingYear;
+            this.hasDraftReturnForYear = hasDraftReturnForYear;
+        }
+
+
+        public object GetFormattedYearText()
+        {
+            return $"{ReportingYear}-{(ReportingYear + 1)%100}";
+        }
+
+        public bool CanChangeScope()
+        {
+            int currentReportingYear = organisation.SectorType.GetAccountingStartDate().Year;
+            int earliestAllowedReportingYear = currentReportingYear - (Global.EditableScopeCount - 1);
+            bool isCurrentYearEditable = ReportingYear >= earliestAllowedReportingYear;
+            return isCurrentYearEditable;

```

Comment:
Literally just personal preference, but I'd find this more readable if it were just
`return ReportingYear >= earliestAllowedReportingYear;`

Thoughts? Happy with either if you find this clearer 😄

Reply:
I'm happy either way 😄